conversion.py

```diff
@@ -1,3 +1,4 @@
 def celsius_to_kelvin(celsius):
-        return celsius + 273.15
-        print('Yay!')
\ No newline at end of file
+    """Convert degrees celsius to kelvin"""
+    return celsius + 273.15
+    print('Yay!')
```

Comment:
```suggestion

```
Oops! I think I had this extra line in my code just to see if the function works. Can this be deleted? :smile: 

Thank you!